### PR TITLE
Give a Background Color to a div Element Guide.md

### PR DESCRIPTION
**Pull request by YuurinBee**
This is a brief description describing two ways of adding a background color to a div Element.

---

## ✅️ By submitting this PR, I have verified the following

- [x] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [x] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [x] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

File: src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md [hunk 1]
```diff
@@ -3,8 +3,37 @@ title: Give a Background Color to a div Element
...
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+You can change the background color to a div Element (or section) by one of two ways.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+<b>The First Method:</b>
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+  Create a class in the style brackets.
+ ```
```
Comment (jonathan-grah) [inline src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md]: Would you mind adding syntax highlighting for html here?

File: src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md [hunk 2]
```diff
@@ -3,8 +3,37 @@ title: Give a Background Color to a div Element
...
+    <style>
+      .blue-background {
+       background-color: blue;
+      }
+    </style>
+ ```
+  You can then add the class to your div Element:
+ ```
```
Comment (jonathan-grah) [inline src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md]: Would you mind adding syntax highlighting for html here?

File: src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md [hunk 3]
```diff
@@ -3,8 +3,37 @@ title: Give a Background Color to a div Element
...
+ 
+ <b>The Second Method:</b>
+ 
+   Instead of creating a class like in the first method, you can create a div Element class in the style brackets.
+    
+   Every div Element will have the class you created and assigned.
+    
+   (This means it is a repeating class for every div Element that you create.)
+
+```
```
Comment (jonathan-grah) [inline src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md]: Would you mind adding syntax highlighting for html here?

File: src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md [hunk 4]
```diff
@@ -3,8 +3,37 @@ title: Give a Background Color to a div Element
...
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+You can change the `background` `color` to a `div` Element (or section) by one of two ways.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+<b>The First Method:</b>
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+  Create a `class` in the style brackets.
+ ```html
+    <style>
```
Comment (jonathan-grah) [inline src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md]: Would you mind removing the unneeded indentation?

File: src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md [hunk 5]
```diff
@@ -3,8 +3,37 @@ title: Give a Background Color to a div Element
...
+      .blue-background {
+       background-color: blue;
+      }
+    </style>
+ ```
+  You can then add the `class` to your `div` Element:
+ ```html
+    <div class="blue-background">
```
Comment (jonathan-grah) [inline src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md]: Would you mind removing the unneeded indentation?

File: src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md [hunk 6]
```diff
@@ -3,8 +3,37 @@ title: Give a Background Color to a div Element
...
+ <b>The Second Method:</b>
+ 
+   Instead of creating a `class` like in the first method, you can create a `div` Element `class` in the style brackets.
+    
+   Every `div` Element will have the `class` you created and assigned.
+    
+   (This means it is a repeating `class` for every `div` Element that you create.)
+
+```html
+    <style>
```
Comment (jonathan-grah) [inline src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md]: Would you mind removing the unneeded indentation?

File: src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md [hunk 7]
```diff
@@ -3,8 +3,37 @@ title: Give a Background Color to a div Element
...
+```html
+<style>
+ .blue-background {
+  background-color: blue;
+  }
+</style>
+```
+You can then add the `class` to your `div` Element:
+```html
+  <div class="blue-background">
```
Comment (jonathan-grah) [inline src/pages/certifications/responsive-web-design/basic-css/give-a-background-color-to-a-div-element/index.md]: Sorry to keep going, but could you just fix the indentation of this code block? Then we will be good to merge 😄

**Comment by ezioda004** [comment]
Thanks @YuurinBee for your contribution. Thanks @jonathan-grah for reviewing!

**Pull request merged**